documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,59 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Create a `GrafanaDatasource` CR so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
```

Comment:
Probably also `cluster-admin` access otherwise you won't be able to install the Grafana operator, assign the `cluster-monitoring-view` role.